src/base/bittorrent/trackerentry.cpp

```diff
@@ -52,6 +53,30 @@ QString TrackerEntry::url() const
     return QString::fromStdString(nativeEntry().url);
 }
 
+QStringList TrackerEntry::messages() const
+{
+    QStringList messages;
+    const auto addMessage = [&messages](const std::string &msg) {
+        QString message = QString::fromStdString(msg).trimmed();
+        if (!message.isEmpty() && !messages.contains(message))
+            messages << std::move(message);
```

Comment:
`QStringList::operator<<` can only take a const reference so `std::move` here is redundant.

Reply:
Exactly, but this will work the same, on if ever that add move operator `<<`  it will start using it without any changes. Hence there is no const also.

Reply:
Right, but currently we don't apply that logic in this codebase, so seeing usages of `std::move()` feels unnatural.

Reply:
Just think of Qt (implicitly shared) types as simple ones (e.g., like `int` etc.).

Reply:
Fixed